aviary/utils/conversion_utils.py

```diff
@@ -0,0 +1,75 @@
+import itertools
+import numpy as np
+
+
+def rep(n, t):
+    """Shorthand for ``itertools.repeat`` with the multiplier first."""
+    return itertools.repeat(t, n)
+
+
+def parse(f, fmt):
```

Comment:
can you change these to `_rep` and `_parse`?
The leading underscore indicates that the function is intended for developers more than users.

Reply:
Okay. Done.